caustic/cosmology/LambdaCDM.py

```diff
@@ -0,0 +1,101 @@
+from .base import Cosmology
+from typing import Any, Optional
+from phytorch.cosmology.drivers.analytic import LambdaCDM as phy_LCDM
+from phytorch.units import si, astro
+from astropy.cosmology import default_cosmology
+
+import torch
+from torch import Tensor
+
+from ..constants import G_over_c2, c_Mpc_s, km_to_Mpc
+from ..parametrized import unpack
+
+__all__ = (
+    "LambdaCDM",
+)
+
+#Default values for parameters, from astropy
+H0_default = float(default_cosmology.get().h)*100.
+Om0_default = float(default_cosmology.get().Om0)
+Ob0_default = float(default_cosmology.get().Ob0)
+Ode0_default = float(default_cosmology.get().Ode0)
+
+class LambdaCDM(Cosmology):
+    """
+    Subclass of Cosmology representing a Lambda Cold Dark Matter (LCDM) cosmology with no radiation.
+    """
+
+    def __init__(
+        self,
+        H0: Optional[Tensor] = torch.tensor(H0_default),
+        Om0: Optional[Tensor] = torch.tensor(Om0_default),
+        Ob0: Optional[Tensor] = torch.tensor(Ob0_default),
+        Ode0: Optional[Tensor] = torch.tensor(Ode0_default),
+        name: str = None,
+    ):
+        """
+        Initialize a new instance of the LambdaCDM class.
+
+        Args:
+            name (str): Name of the cosmology.
+            H0 (Optional[Tensor]): Hubble constant over 100. Default is H0_default from Astropy.
+            Om0 (Optional[Tensor]): Matter density parameter at z=0. Default is Om0_default from Astropy.
+            Ob0 (Optional[Tensor]): Baryon density parameter at z=0. Default is Ob0_default from Astropy.
+            Ode0 (Optional[Tensor]): Dark Energy density parameter at z=0. Default is Ode0_default from Astropy.
+        """
+        super().__init__(name = name)
+
+        self.add_param("H0", H0)
+        self.add_param("Om0", Om0)
+        self.add_param("Ob0", Ob0)
+        self.add_param("Ode0", Ode0)
+        self.cosmo = phy_LCDM(H0 = H0, Om0 = Om0, Ob0 = Ob0, Ode0 = Ode0)
+
+    def hubble_distance(self, H0):
```

Comment:
Should this be `H0` or `h0`?

Reply:
Big H0...phytorch uses that as the convention rather than little h. While it is a break from the FlatLambdaCDM way, it is in line with the astropy convention which is what most people are used to.

Reply:
Perhaps we should change FlatLambdaCDM to also use big H0 for consistency?

Reply:
Yeah, that would be better

Reply:
Fixed. I also noticed that whoever wrote the tests before, was using big H0 = little h when instantiating cosmologies from astropy and then dividing by 100, which is a rather janky way of doing things. So I am glad we have switched to Big H0.